app/controllers/liabilities_controller.rb

```diff
@@ -0,0 +1,27 @@
+class LiabilitiesController < ApplicationController
+  def pay
+    operation = CreatePayment.new(params[:id])
+
+    target = unless operation.error?
```

Comment:
Elsewhere in the code, the pattern seems to be;
```
    if some_error?
      response = error_reponse
    else
      response = its_all_good
    end
````
So, for consistency and readability, I think it we could use the same pattern here, i.e.;
````
    if operation.error?
      target = root_path
      flash[:alert] = t('.govpay_api_error')
    else
      target = operation.payment_url
    end
````